fh-dataman.json

```diff
@@ -0,0 +1,238 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1.0.0",
+  "metadata": {
+    "name": "fh-dataman",
+    "annotations": {
+      "description": "Red Hat Mobile Databrowser Backend Service template",
+      "iconClass": "icon-nodejs"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "fh-dataman-service",
+        "labels": {
+          "name": "fh-dataman-service"
+        },
+        "annotations": {
+          "service.alpha.openshift.io/dependencies": "[{\"name\":\"mongodb-1\",\"namespace\":\"\",\"kind\":\"Service\"},{\"name\":\"fh-messaging-service\",\"namespace\":\"\",\"kind\":\"Service\"},{\"name\":\"fh-metrics-service\",\"namespace\":\"\",\"kind\":\"Service\"},{\"name\":\"fh-statsd-service\",\"namespace\":\"\",\"kind\":\"Service\"}]"
+        }
+      },
+      "spec": {
+        "ports": [
+          {
+            "port": 8080
+          }
+        ],
+        "selector": {
+          "name": "fh-dataman"
+        }
+      }
+    },
+    {
+      "kind": "DeploymentConfig",
+      "apiVersion": "v1.0.0",
+      "metadata": {
+        "name": "fh-dataman",
+        "labels": {
+          "name": "fh-dataman"
+        }
+      },
+      "spec": {
+        "strategy": {
+          "type": "Rolling",
+          "resources": {
+            "limits": {
+              "cpu": "800m",
+              "memory": "800Mi"
+            },
+            "requests": {
+              "cpu": "200m",
+              "memory": "200Mi"
+            }
+          }
+        },
+        "triggers": [
+          {
+            "type": "ConfigChange"
+          }
+        ],
+        "replicas": "${FH_DATAMAN_REPLICAS}",
+        "selector": {
+          "name": "fh-dataman"
+        },
+        "template": {
+          "metadata": {
+            "labels": {
+              "name": "fh-dataman"
+            }
+          },
+          "spec": {
+            "containers": [
+              {
+                "name": "fh-dataman",
+                "image": "${FH_DATAMAN_IMAGE}:${FH_DATAMAN_IMAGE_VERSION}",
+                "ports": [
+                  {
+                    "containerPort": "${FH_DATAMAN_PORT}"
+                  }
+                ],
+                "env": [
+                  {
+                    "name": "FH_DATAMAN_NAMESPACE",
```

Comment:
The name of the OpenShift project the app is deployed into. Populated from the Kubernetes downward api https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information. Might not be required unless used by your app